### PR TITLE
Tests for accounts, user and session controller

### PR DESCRIPTION
**Pull request by ivanmauricio**
Add test for accounts, user and session controller

### CODE REVIEW

File: test/support/test_helpers.ex [hunk 1]
```diff
@@ -0,0 +1,18 @@
+defmodule ActionForChildren.Web.TestHelpers do
```
Comment (andrewMacmurray) [inline test/support/test_helpers.ex]: is there a reason you've put these in a module? They look like an exact copy of what's in the `Accounts` module

Comment (ivanmauricio) [inline test/support/test_helpers.ex]: It seemed wrong to use functions from a module being tested in its own tests. Then I remembered that this is the pattern they use in the Phoenix book. It made the tests a lot cleaner to write.

Comment (ivanmauricio) [inline test/support/test_helpers.ex]: What do you think?

Comment (andrewMacmurray) [inline test/support/test_helpers.ex]: fair enough, in that case could you change the other bits that use the `Accounts` module in the tests so we're consistent? I think they're in `page`, `user` and `auth`

Comment (ivanmauricio) [inline test/support/test_helpers.ex]: Will do!

**Pull request merged**